src/main/java/com/nerodesk/App.java

```diff
@@ -100,113 +102,121 @@ public App(final Base base) throws IOException {
-                                public void run() {
-                                    new VerboseProcess(
-                                        new ProcessBuilder(
-                                            "mvn",
-                                            "sass:update-stylesheets",
-                                            "minify:minify"
-                                        )
-                                    ).stdout();
+                                public Take route(final Request req)
+                                    throws IOException {
+                                    return new TkDocs(
+                                        base.user(
+                                            new RqAuth(req).identity().urn()
+                                        ).docs(),
+                                        req
+                                    );
                                 }
-                            },
-                            new TsFiles("./target/classes")
+                            }
                         ),
-                        new FkFixed(new TsClasspath())
-                    ),
-                    "text/css"
-                )
-            ),
-            new FkRegex("/robots.txt", ""),
-            new FkRegex(
-                "/",
-                new TsFork(
-                    new FkAuthenticated(
-                        new Target<Request>() {
-                            @Override
-                            public Take route(final Request req)
-                                throws IOException {
-                                return new TkDocs(
-                                    base.user(
-                                        new RqAuth(req).identity().urn()
-                                    ).docs(),
-                                    req
-                                );
+                        new FkAnonymous(
+                            new Target<Request>() {
+                                @Override
+                                public Take route(final Request req) {
+                                    return new TkIndex(req);
+                                }
                             }
+                        )
+                    )
+                ),
+                new FkRegex(
+                    "/r",
+                    new Takes() {
+                        @Override
+                        public Take route(final Request req)
+                            throws IOException {
+                            final String file =
+                                new RqHref(req).href()
+                                    .param("f").iterator().next();
+                            return new TkRead(
+                                base.user(
+                                    new RqAuth(req).identity().urn()
+                                ).docs().doc(file)
+                            );
```

Comment:
@longtimeago this `FkRegex` looks pretty similar to the next one except `"/r"` and `"f"`, maybe we can reuse it somehow?